source/_patterns/20-molecules/navigation/highlight-nav.twig

```diff
@@ -0,0 +1,14 @@
+<div class="highlight-nav">
+    <span class="highlight-nav-title">Schwerpunkte</span>
```

Comment:
Ein versteckter Titel analog "Hauptnavigation" in der Global-Nav wäre für die Accessibility gut

Reply:
hat's doch schon, nöd?